crates/sui-graphql-rpc/src/types/move_registry/named_move_package.rs

```diff
@@ -23,8 +23,8 @@ use super::{
 pub(crate) struct NamedMovePackage;
 
 impl NamedMovePackage {
-    /// Queries a package by name (and version, encoded in the name).
-    /// Name's format should be `{application}@{organization}/v{versiion}`.
+    /// Queries a package by name (and version, encoded in the name but optional).
+    /// Name's format should be `{organization}/{application}:v{versiion}`.
```

Comment:
```suggestion
    /// Name's format should be `{organization}/{application}:v{version}`.
```

Reply:
A vote maybe against `:` is the fact that it may be visually confusing next to a fully-qualified name separator:

```move
@acme/widget:v2::foo::bar
@acme/widget.v2::foo::bar
@acme/widget/v2::foo::bar

@acme/widget:2::foo::bar
@acme/widget.2::foo::bar
@acme/widget/2::foo::bar
```

Reply:
Generally, even though we allow it, we'd always encourage to not include versions in Types (they don't really offer anything), so I am not really concerned about this scenario! thoughts?